examples/length/length.description

```diff
@@ -1,2 +1,2 @@
-// Ballerina supports the `length()` builtin function that returns the length/size of the given value.
+// Ballerina supports the builtin function `length()`  that returns the length/size of the given value.
```

Comment:
shouldn't this be method ? IINM, we say built-in method right? 

https://docs.google.com/presentation/d/1ZnykKSai8WFJzT6GMIuj1s-6bMwGYaawm_Q1E4_PBa4/edit#slide=id.g424ccecdb3_0_55